infrastructure/configs/config.go

```diff
@@ -56,7 +57,13 @@ func (p Phase) String() string {
 	return "local"
 }
 
+var instance *Config
```

Comment:
Since NewConfig is only called once in `server.go` with wire, this instance variable and related code seems unnecessary.

Reply:
@minkj1992 
As I understand it, isn't instantiation happening once more here?
similarly, since the config instance is not used as a global variable, I thought if injection using wire would continue to be created.

- https://github.com/Nexters/myply/blob/32aa3616efa2d5329a51d7221aaed5b1553f9925/application/server.go#L33-L35

Reply:
Oh I see!
Since wire creates and injects dependencies through wire.build() (https://github.com/Nexters/myply/blob/32aa3616efa2d5329a51d7221aaed5b1553f9925/application/wire_gen.go#L50)

Unless it is really necessary, we should call wire.build() only once on the server. except we build microservice that has multiple server instances!